src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &--1 :first-child,
+  &--2 :nth-child(-n + 2),
+  &--3 :nth-child(-n + 3),
+  &--4 :nth-child(-n + 4),
+  &--5 :nth-child(-n + 5) {
+    background-image: url(/src/images/star-active.svg);
+  }
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  }
+
+  &__star:not(:first-child) {
+    margin-left: 0;
```

Comment:
The `margin-left: 0;` style in `&__star:not(:first-child)` seems unnecessary as there is already a `margin-right: 4px;` specified for `&__star`. If this is intentional to override some other styles, please ensure that it's needed, otherwise, it could be removed to clean up the code.